pallets/proofs-dealer/runtime-api/src/lib.rs

```diff
@@ -7,13 +7,14 @@ use sp_std::vec::Vec;
 
 sp_api::decl_runtime_apis! {
     #[api_version(1)]
-    pub trait ProofsDealerApi<ProviderId, BlockNumber, Key, RandomnessOutput, TrieRemoveMutation>
+    pub trait ProofsDealerApi<ProviderId, BlockNumber, Key, RandomnessOutput, TrieRemoveMutation, Balance>
```

Comment:
This seems to be unnecessary. I assume it's a leftover from when you were adding the runtime API here.